rs/src/parse/expressions/literals.rs

```diff
@@ -1049,6 +1091,26 @@ fn parse_null(x: &substrait::Type, y: &mut context::Context) -> diagnostic::Resu
     }
 }
 
+fn parse_value(
+    x: &substrait::expression::literal::user_defined::Val,
+    y: &mut context::Context,
```

Comment:
ctx?

Reply:
Yes, can do. The rest of this file uses `x` and `y`, but `ctx` is better 👍